docs/deployment-workflow.md

```diff
@@ -0,0 +1,177 @@
+# Git Branching Strategy
+
+## Overview
+
+This document outlines our Git branching strategy for managing staging and production deployments. It provides guidelines for branch management, feature development, releases, and hotfixes.
+
+Our strategy is designed to support:
+- Bi-weekly/tri-weekly production deployments
```

Comment:
How come this bi-weekly/tri-weekly production deployment? We could deploy after each sprint (4 weeks)